### DISCUSSION TITLE
Application Lists / Categories

### DISCUSSION THREAD

**Discussion by eneller** [Ulauncher/Ulauncher#1204] [General]
I just wanted to ask if there currently is some way to create application lists (categories of applications) similar to the ones in the xfce (whisker-menu) and most other gui-focused launchers?
Or would the preferred way be to create a custom extension?

I just sometimes forget the name of a tool i have installed but use infrequently.

**Comment by friday** [comment]
I'm not sure I understand. There's no menu. The intention of Launchers such as Ulauncher, Albert, etc is to be able to use just your keyboard and launch an app fast.

Some of the metadata such as description and tags are searchable in Ulauncher v6 (not yet released).

**Reply by eneller** [reply]
Well, the better question probably would be the following:
What would be the preferred way of achieving this tree-style menu in ulauncher?
Currently, plugins only work as prefixes, so there would be one prefix per "folder" (application group).
Do you think it could be desirable to just add entries to ulauncher that have custom behavior?
Or is ulauncher strictly single-purpose and would reject such features?

**Reply by friday** [reply]
> Or is ulauncher strictly single-purpose and would reject such features?

Menu launchers are a different suit of applications for people who want that to click their way through a hierarchy rather than use the more productive keyboard focused ways. Feel free to use a menu launcher instead of Ulauncher, or make an extension.
